Makefile

```diff
@@ -0,0 +1,19 @@
+
+local: test typecheck flake8 pylint
+
+remote: test-cov typecheck flake8 pylint
```

Comment:
In my opinion, the **remote** Makefile goal should run the whole testsuite.

Reply:
That's what it's doing, the only difference is that tests are executed with the coverage flags.

Reply:
Oh, didn't notices the _pytest_ in the test-cov goal. Cheers!